test/unit/jasmine-boot.js

```diff
@@ -41,9 +41,8 @@
 'use strict';
 
 function initializePDFJS(callback) {
-  require.config({paths: {'pdfjs': '../../src', 'pdfjs-web': '../../web',
-                 'pdfjs-test': '..'}});
-  require(['pdfjs/display/global', 'pdfjs-test/unit/annotation_spec',
+    Promise.all([
+           'pdfjs/display/global', 'pdfjs-test/unit/annotation_spec',
```

Comment:
The indentation is inconsistent with the rest of the files. I'd prefer if we keep the same indentation as we do for other files here as well, i.e., indent with two spaces (I don't mind the diff being a few lines larger).